source/blog/20019-01-22-sso-keycloak-ovirt-openidc-389ds.html.md

```diff
@@ -246,3 +248,20 @@ connection.close()
 
 Login to webadmin, user1 belongs to group ovirt-administrator in ldap, so should get access webadmin automatically.
```

Comment:
"so should get access webadmin automatically" -> "so it should be able to login to webadmin without any additional permission settings"